suite/suite_test.go

```diff
@@ -512,3 +515,75 @@ func TestSuiteWithStats(t *testing.T) {
 	assert.NotZero(t, testStats.End)
 	assert.True(t, testStats.Passed)
 }
+
+// Suite to test behavior when running with the failfast flag
```

Comment:
One small nit, could we have this docstring follow standard Go convention for exported funcs and types (i.e it starts with the object name, `FailfastSuite`) ?

Reply:
No problem. Done :)